src/miniscript/types/extra_props.rs

```diff
@@ -851,20 +853,19 @@ impl Property for ExtData {
                 Ok(Self::from_multi(k, pks.len()))
             }
             Terminal::After(t) => {
-                // FIXME check if t > 2^31 - 1
-                if t == 0 {
+                if t == 0 || (t & SEQUENCE_LOCKTIME_DISABLE_FLAG) == 1 {
```

Comment:
I don't think that there are any rules applicable for after fragment. The disable flag is only for CSV.

Reply:
Right, i was confused by the FIXME comment and didn't double check.. FIxed, thanks!

Reply:
Reverted post discussion on IRC and added a rationale comment so we don't forget about it :)